requirements-base.txt

```diff
@@ -47,6 +47,7 @@ oauth2>=1.5.167
 parsimonious==0.8.0
 percy>=1.1.2
 petname>=2.6,<2.7
+phonenumberslite<8.0
```

Comment:
You're gonna want to add an `==` pin for whatever `<8.0` resolves to in getsentry.

Reply:
@joshuarli fixed

Reply:
You did it in the wrong repo. getsentry == pins, sentry range pins. The `<8.0` is correct here, just put that == pin in your other pr: https://github.com/getsentry/getsentry/pull/3377

Reply:
Oh oops, I assumed that no requirements diff in the getsentry pr means you didn't add it, but it's already there because we install twilio in prod by default. You're good then.